css/styles.css

```diff
@@ -1,413 +1,538 @@
+.card_content {
+  margin-top: 8rem;
 }
-#tags{
-    display: flex;
-    flex-direction: row;
-    justify-content: left;
-    margin-bottom: 2rem;
+
+.card_content > h4 {
+  font-weight: bold;
 }
-#tags li{
-    background-color: rgba(255, 255, 255, 0.24);
-    padding: 8px;
-    color: white;
-    margin-right: 5px;
+
+#tags {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+  margin-bottom: 2rem;
 }
 
-/* About me Section*/
-#About{
-    background-color: #262626;
-    height: fit-content;
-    padding: 8rem 1rem;
-    position: relative;
-    overflow: hidden;
+#tags li {
+  background-color: rgba(255, 255, 255, 0.24);
+  padding: 8px;
+  margin-right: 5px;
+}
+
+/* ABOUT SECTION */
+#About {
+  background-color: #262626;
+  height: fit-content;
+  padding: 8.13rem 1rem;
+  overflow: hidden;
+}
+
+.about_text {
+  width: 100%;
+}
+
+#resume_button {
+  margin-top: 2rem;
+}
+
+#about_image {
+  margin-left: 50%;
+}
+
+#skills {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+#skills h4 {
+  font-weight: bold;
+}
+
+#skills ul {
+  margin-bottom: 2rem;
 }
 
-.resume-button{
-    background-color:#F55800;
-    padding: 8px;
-    color: white;
-    margin-top: 3rem;
-    display: block;
-    width:fit-content;
+#skills li {
+  margin-bottom: 2rem;
+  text-align: center;
 }
 
-#skills{
-    margin-top: 20%;
+.line {
+  display: block;
+  width: 100%;
+  margin: 8rem 0;
+  border-bottom: solid 2px white;
+}
+
+@media (min-width: 768px) {
+  #About {
+    padding: 8.13rem 4rem;
+  }
+
+  .about_container {
     display: flex;
     flex-direction: row;
-    flex-wrap: wrap;
-    justify-content: space-around;
+  }
+
+  .about_text {
+    width: 50%;
+  }
+
+  #about_image {
+    width: 50vw;
+    margin-right: -15%;
+    margin-left: 1rem;
+  }
+
+  #skills {
+    flex-direction: row;
+    align-content: center;
+    text-align: center;
+    margin-top: 8rem;
+  }
 }
-#skills h3{
-    font-size: 2rem;
+
+/* CONTACT ME SECTION */
+
+.contact_container {
+  padding: 8.13rem 1rem;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 2rem;
 }
-#skills li{
-    color:white;
-    margin-bottom: 2rem;
-    font-size: 1.5rem;
-    text-align: center;
+
+.contact_container h3 {
+  text-align: center;
+}
+
+#contact_form {
+  display: flex;
+  flex-direction: column;
+  gap: 1.5rem;
+  margin-top: 3rem;
+}
+
+#contact_form * {
+  font-size: 1rem;
+  border: none;
+  border-bottom: solid 1px #979493;
+  padding: 1rem;
+}
+
+#text_box {
+  border: none;
+  background-color: #fbf8f7;
+  height: 200px;
+}
+
+#contact_button {
+  background-color: #f55800;
+  padding: 0.75rem;
+  font-weight: bold;
+  display: block;
+  width: fit-content;
+  border-style: none;
+  margin-inline: auto;
+  font-size: var(--button_font_size);
+  color: var(--button_font_color);
+}
+
+#contact_button:hover {
+  background-color: white;
+  border: solid 2px #ff8843;
+  cursor: pointer;
+}
```

Comment:
- [ ] kindly consider giving this button color so as the text it contains will keep being visible when a person hovers it
- your design 👇 
![15](https://user-images.githubusercontent.com/60146030/221004460-7d57285b-fb18-4574-9fe4-4b9cb929260f.PNG)
- Figma design 👇 
![12](https://user-images.githubusercontent.com/60146030/221004552-456d2a43-71ea-488b-8e54-fa8829539966.PNG)